spec/common_scripting_layer/histogram_spec.rb

```diff
@@ -0,0 +1,45 @@
+context 'Histogram' do
+  it 'Sets up a basic reference image for histogram with random numbers' do
+    plot = Rubyplot::Histogram.new(600)
+    plot.title = 'Random Histogram Numbers'
+    plot.marker_count = 8
+    plot.data('data', [5, 12, 9, 6, 7])
+    plot.write('spec/reference_images/histogram_test_1.png')
+
+    expect(compare_with_reference?('histogram.png', 'histogram_test_1.png', 10)).to eq(true)
+  end
+
+  it 'Fails to match with the reference image' do
```

Comment:
Okay, so this one is grammatically correct (except for the capitalization), but doesn't actually say *why* it should fail to match with the reference image. Does this demonstrate a false comparison?

Reply:
Here is the difference look closely.  

1. plot.data('data', [5, 12, 9, 6, 7])
2. plot.data('data', [5, 12, 9, 6, 6])

This difference is a very subtle difference that is used to observe the failure.

Reply:
You are misunderstanding my critique. Your spec titles should _say_ what the difference is. People shouldn't have to search through your spec to find it.